### DISCUSSION TITLE
Is there a PDF available

### DISCUSSION THREAD

**Discussion by FFABrown** [Ghostkeeper/SettingsGuide#114] [Q&A]
Hi, I was wondering if a PDF version of the information was available. I use this plugin all the time in Cura, but would love to be able to read it outside my computer. Thank you, and it doesn't have to be PDF, just anything I would be able to print and keep next to my 3D printer.

**Comment by Ghostkeeper** [comment]
Not as far as I know. Perhaps there could be a way to convert Markdown files automatically to PDF or to print form? Or to HTML, and then list all of them after one another, to print using a browser? The Settings Guide itself uses a Python library to convert the Markdown to a HTML subset for Qt.

You might find some difficulties with automatic processes though if you want to do it proper. In particular, many articles share the same images and printing them for each would be quite wasteful (e.g. with all the speed settings). And of course, animations won't work in printed form. The guide also has a bit of customisation for how it displays the images with their headers that would need to be re-implemented.
